docs/template-helpRequest.md

```diff
@@ -0,0 +1,66 @@
+**Help Request**
+
+**Issue Title:** Help Request: [summary of your problem]
+
+
+**Operating System**
+
+
+**JDK Version**
+
+
+**Eclipse Version**
+
+
+**Firefox / Chrome Version (where applicable)**
+
+
+**Step in the `settingUp.md` Guide / Failing Tests**
+
+*If you are setting up the project:*
+> Tell us the step in the setting up guide at which you are stuck
+
+*If you are resolving test failures:*
+> Tell us which tests are failing
+
+*Miscellaneous*
+> Tell us what you were trying to to / what you need help with
+
+**Error Messages Shown...**
+
+*...in the TestNG tab, if you are resolving test failures:*
+> e.g. Failed tests:
+> parallel-ui-tests -> teammates.test.cases.ui.browsertests.InstructorFeedbackResultsPageUITests
+> Stack Trace (or screenshot):
+> java.lang.AssertionError:
+> 	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
+> 	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
+> 	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:267)
+> 	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:259)
+> 	at teammates.test.cases.BaseTestCase.assertNotNull(BaseTestCase.java:177)
+> 	at
+>   teammates.test.cases.action.AdminAccountDeletePageActionTest.testExecuteAndPostProcess(AdminAccountDeletePageActionTest.java:42)
```

Comment:
This looks quite ugly when converted to markdown. It should be formatted as code instead, like:
```
java.lang.AssertionError:
 	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
 	at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
 	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:267)
 	at org.testng.AssertJUnit.assertNotNull(AssertJUnit.java:259)
 	at teammates.test.cases.BaseTestCase.assertNotNull(BaseTestCase.java:177)
 	at teammates.test.cases.action.AdminAccountDeletePageActionTest.testExecuteAndPostProcess(AdminAccountDeletePageActionTest.java:42)
```